core/client/verify.go

```diff
@@ -38,26 +38,26 @@ var (
 	ErrNilProof = errors.New("nil proof")
 )
 
-// Verifier is a client helper library for verifying request and responses.
-type Verifier struct {
-	vrf         vrf.PublicKey
-	mapVerifier *tclient.MapVerifier
-	logVerifier *tclient.LogVerifier
+// Verify is a client helper library for verifying request and responses.
+type Verify struct {
```

Comment:
Is there a way to name this with a noun? Some ideas:
- Call this `Verifier` and the interface - `IVerifier`. Not sure whether Go naming conventions allows this.
- `Verification(Client)`, `VerifierImpl`, `VerifiedClient` ...

Reply:
I struggled with the naming conventions too. 
is `RealVerifier` appropriate?  The interface exists so we can have a `FakeVerifier`